lib/Command/CleanUp.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2022 Daniel Kesselberg <mail@danielkesselberg.de>
+ *
+ * @author Daniel Kesselberg <mail@danielkesselberg.de>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\TwoFactorWebauthn\Command;
+
+use OCA\TwoFactorWebauthn\Db\PublicKeyCredentialEntityMapper;
+use OCP\DB\Exception;
+use OCP\IDBConnection;
+use OCP\IUserManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CleanUp extends Command {
+	/** @var IDBConnection */
+	private $db;
+
+	/** @var IUserManager */
+	private $userManager;
+
+	/** @var PublicKeyCredentialEntityMapper */
+	private $webauthnMapper;
+
+	public function __construct(
+		IDBConnection $db,
+		IUserManager $userManager,
+		PublicKeyCredentialEntityMapper $webauthnMapper
+	) {
+		parent::__construct();
+
+		$this->db = $db;
+		$this->userManager = $userManager;
+		$this->webauthnMapper = $webauthnMapper;
+	}
+
+	protected function configure(): void {
+		$this->setName('twofactor_webauthn:cleanup');
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		$io = new SymfonyStyle($input, $output);
+		$io->title('Find orphaned webauthn credentials');
+		$io->text([
+			'This command search webauthn credentials for deleted users.',
+			'Found credentials for deleted users will be removed.'
+		]);
+
+		$confirm = $io->confirm('Start process?', true);
```

Comment:
do we really need this? :see_no_evil: 

I'd say if the admin calls this command they should know about the consequences

Reply:
> should know about the consequences

Good point indeed. I forgot to add a description for the command :see_no_evil: 

> do we really need this? see_no_evil

I don't have a opinion :man_shrugging: Do you want me to remove it?

Reply:
Confirm is gone.